auto3dseg/algorithm_templates/segresnet/configs/hyper_parameters.yaml

```diff
@@ -23,6 +23,7 @@ cache_rate: null
 determ: false
 fold: 0
 learning_rate: 2.0e-4
+batch_size: "$@num_images_per_batch"
```

Comment:
you don't need a dollar $ sign here, I think it should be simply @num_images_per_batch without quotes, but please test

Reply:
PS: alsowe should agree on common keys ( in the future) such batch_size, warmup_epochs, etc, so that we don't need to create aliases

Reply:
thanks! just checked. not working.

it is better to unify the naming soon.